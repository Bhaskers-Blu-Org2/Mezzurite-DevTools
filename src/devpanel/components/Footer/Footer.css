.Footer {
    /* height: 50px; */
    background-color: #363b44;
    color: #f9bb10;
    flex-shrink: 0;
    padding: 10px;
}