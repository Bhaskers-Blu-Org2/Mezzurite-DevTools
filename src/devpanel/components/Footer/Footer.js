import React from 'react';
import { string } from 'prop-types';

import './Footer.css';
import GithubMark from './GithubMark';

const Footer = (props) => (
  <footer role='contentinfo'>
    <ul className='footer--content'>
      <li className='github-information'>
        <a
          aria-label='Visit Mezzurite Dev Tools on GitHub'
          className='github-information--link'
          href='https://github.com/Microsoft/Mezzurite-DevTools'
          target='_blank'
        >
          <GithubMark />
          <span className='github-information--text'>Visit us on GitHub!<span className='tablet-hidden'> ~ The Mezzurite Development Team</span></span>
        </a>
      </li>
      <li className='package-information'>
        <ul className='package-information--container'>
          {props.packageName != null && <li className='package-information--name'>
            <span className='package-information--label'><span className='tablet-hidden'>Package </span>Name:</span>
            <a aria-label='Visit Mezzurite on GitHub' className='github-information--package' href='https://github.com/Microsoft/Mezzurite' target='_blank'>
              {props.packageName}
            </a>
          </li>}
          {props.packageVersion != null && <li className='package-information--version'>
            <span className='package-information--label'><span className='tablet-hidden'>Package </span>Version:</span>{props.packageVersion}
          </li>}
        </ul>
      </li>
    </ul>
  </footer>
);

Footer.propTypes = {
  packageName: string,
  packageVersion: string
};

export default Footer;
