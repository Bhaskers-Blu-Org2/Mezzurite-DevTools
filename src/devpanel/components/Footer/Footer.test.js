import Footer from './Footer';

describe('Footer.js', () => {
  it('should render the footer', () => {
    expect(Footer()).toMatchSnapshot();
  });
});
