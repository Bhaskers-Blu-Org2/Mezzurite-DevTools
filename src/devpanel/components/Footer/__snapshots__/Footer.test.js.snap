// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.js should render the footer 1`] = `
<footer
  className="Footer"
>
  <a
    href="https://github.com/Microsoft/Mezzurite-DevTools"
    target="_blank"
  >
    <img
      id="gh-logo"
      src="test-file-stub"
    />
  </a>
  <span>
    Visit us on GitHub! ~ The Mezzurite Development Team
  </span>
</footer>
`;
